envs/node/transport.js

```diff
@@ -9,10 +9,24 @@ class Transport extends Obfuscated {
     this.dc = dc;
     this.debug = baseDebug.extend(`transport-${this.dc.id}`);
     this.crypto = crypto;
+    this.destroyed = false
+    this.handleConnect = this.handleConnect.bind(this)
+    this.handleData = this.handleData.bind(this)
+    this.handleError = this.handleError.bind(this)
+    this.handleClose = this.handleClose.bind(this)
```

Comment:
Please stick to the style. Add a commas and semicolons

Reply:
fixed here https://github.com/alik0211/mtproto-core/pull/235/commits/185d52e3ffbdc65bcfe7b69d810a0b4a4ff87446